packages/appstore/src/components/add-more-wallets/wallet-add-card.tsx

```diff
@@ -31,7 +30,16 @@ const AddWalletCard = observer(({ wallet_info }: TAddWalletCard) => {
     return (
         <div className='add-wallets__card'>
             <div className='add-wallets__card-wrapper'>
-                <WalletCard wallet={wallet_details} size='medium' state={is_added ? 'added' : 'add'} />
+                <WalletCard
+                    wallet={wallet_details}
+                    size='medium'
+                    state={is_added ? 'added' : 'add'}
+                    onClick={() => {
+                        if (!is_added) {
+                            createWallet({ payload: { currency, account_type: 'crypto' } });
+                        }
+                    }}
```

Comment:
`nit-pick:` perhaps to make it one line we could it like this?
```suggestion
                    onClick={() => !is_added && createWallet({ payload: { currency, account_type: 'crypto' } })}
```